web/static/js/data/api.js

```diff
@@ -0,0 +1,19 @@
+var $ = require('jquery');
```

Comment:
I suggest to use superagent to do ajax requests: 
http://visionmedia.github.io/superagent/ 
Jquery is a bit too heavy for this sole task!